docs/source/actions.rst

```diff
@@ -183,6 +183,8 @@ A notification will look like the following one:
    :scale: 50 %
    :alt: Slack Notification
 
+`Follow these steps`_ for creating a bot in your workplace.
```

Comment:
Looking at the paragraph that starts this section, “This action allows you to […]”, I think that this paragraph here should come before the documentation describes the need to provide the bot credentials.

What would you think about merging the starting paragraph and your new paragraph here as follows?

```
This action allows you to send custom messages to a `Slack`_ channel or user
when your monitor suites finishes their execution.

To use this action you need to `create a Slack bot account`_ and provide the
`Slack credentials`_ in your `settings.py` file as follows:
```